Gemfile.local.example

```diff
@@ -27,6 +27,9 @@ end
 
 # Create a custom group
 group :local do
-  # Add the lab gem so that the 'lab' plugin will work again
+  # This is the first way to add a non-standard gem file dependency in.
   gem 'lab', '~> 0.2.7'
+  # And this is another way that references local directories to find and compile the gem file as needed.
+  # This is the optimal method for testing Gem PRs such as those in rex-text or rex-powershell.
+  gem 'rex-powershell', path: '/home/gwillcox-r7/git/rex-powershell'
```

Comment:
```suggestion
  gem 'rex-powershell', path: '../rex-powershell'
```

Reply:
I think this may need an absolute path? Or at least every example that I've seen so far has used that? Happy to change this if you think this is better though, the example was just to showcase to users how they might use it.

Reply:
After further investigation I have confirmed that you can actually use relative paths. Sorry for the confusion here, let me get this updated so long 👍

Reply:
Recommended fix has been applied, marking this as resolved.